pkg/renderer/readme.go

```diff
@@ -47,6 +47,8 @@ the login page, and log in with username "admin" and the password found in the
   - Deploy Scanner components
     - Run scanner/scripts/setup.sh
     - Run {{.K8sConfig.Command}} create -R -f scanner
+
+  - Optional: Deploy Scanner components
```

Comment:
```suggestion
  - Optional: Deploy Scanner V4 components
```

Otherwise it would be somewhat repetitive, compared with l47.

Reply:
That's what I meant to write 🤦

Reply:
I guess you just wanted to test me!